test/tests/core.json

```diff
@@ -0,0 +1,415 @@
+{
+  "scenario": "Core syntax",
```

Comment:
consider naming this file `syntax.json` and avoiding the term "core". Are there "non-core" syntax elements?

Reply:
Sounds good 👍  I think we're unlikely to have "non-core".